airsonic-main/pom.xml

```diff
@@ -538,19 +538,19 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>42.1.4</version>
+            <version>42.2.5</version>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.43</version>
+            <version>8.0.16</version>
```

Comment:
Wow, skipping 2 majors version!

Reply:
Yeah I imagine this one is gonna require someone with mysql to test before it gets merged.